pkg/integration/replication/synchronous_replication_test.go

```diff
@@ -21,13 +21,9 @@ func TestSyncTestSuite(t *testing.T) {
 	suite.Run(t, &SyncTestSuite{})
 }
 
-// this function executes before the test suite begins execution
-func (suite *SyncTestSuite) SetupSuite() {
-	suite.baseReplicationTestSuite.SetupSuite()
+func (suite *SyncTestSuite) TestSyncFromMasterToAllFollowers() {
 	suite.SetupCluster(2, 2)
```

Comment:
This could be a problem with docker, with so many containers being spawned and closed for a test. If we want many followers, we can create diff test suites right? For ex, for TestMinimumFollowers, we can add a testsuite MinFollowerTestSuite. I think this function would be better in SetupSuite

Reply:
ACK, I agree that this could affect the performance.

Although that way we don't have a clean-page start for each test (like there could already be some transactions in the database). This can be fixed by creating new DB for each test later.

Reply:
in that scenario, this can be added to BeforeTest right?

Reply:
I think suite has a BeforeTest and AfterTest which executes before/after each test

Reply:
Fixed